README.md

```diff
@@ -22,29 +22,66 @@
-* **Integrated with git.** Run data validation tasks as part of your CI/CD process to test changes everywhere you data is used. No coordination meetings required.
+* **Centralized data tests.** Write data validation tests which run whenever upstream data sources change.
+* **Integrated with git.** Run data validation tasks as part of your CI/CD process to test changes everywhere you data is used.
 * **Your data, your cloud.** Grai is fully open sourced and self hosted. You maintain full control over your data and hosting environment.
 
 
-## Quick Start
+## How it works
+
+* Automatically build column level lineage spanning your warehouse and production services with integrations for `dbt`, `snowflake`, `fivetran` and more (see below).
+* Get alerts in your CI/CD workflows whenever changes to a production system will impact your warehouse or DBT projects with [github actions](https://github.com/grai-io/grai-actions-server).
+* Self host the project or run it in the [cloud](https://app.grai.io) for free.
+
+### Connectors
+
+We currently support pre-built connectors for a variety of sources including
+
+
+<center>
 
-A great place to start is our Quick Start guide in our [documentation](https://docs.grai.io/quick-start). To get going very quickly
+|            |  integrations  | install                              |
+| ---------- |  ------------  | ------------------------------------ |
+| <img src="resources/snowflake.png" alt= “” width="30" height="30"> | Snowflake  | `pip install grai-source-snowflake`  |
+| <img src="resources/bigquery.svg" alt= “” width="30" height="30">  | BigQuery   | `pip install grai-source-bigquery`   |
+| <img src="resources/postgres.png" alt= “” width="30" height="30">  | Postgres   | `pip install grai-source-postgres`   |
+| <img src="resources/mysql.png" alt= “” width="30" height="30">     | MySQL      | `pip install grai-source-mysql`      |
+| <img src="resources/mssql.png" alt= “” width="30" height="30">     | SQL Server | `pip install grai-source-mssql`      |
+| <img src="resources/dbt.png" alt= “” width="30" height="30">       | dbt        | `pip install grai-source-dbt`        |
+| <img src="resources/fivetran.png" alt= “” width="30" height="30">  | Fivetran   | `pip install grai-source-fivetran`   |
+| <img src="resources/csv.png" alt= “” width="30" height="30">       | csv        | `pip install grai-source-flat-file`  |
+
+</center>
+
+## Quickstart
+
+You can find a full quickstart guide in the [documentation](https://docs.grai.io/quick-start) which covers deploying your own instance of Grai and getting set up with your first connector in python.
+
+
+### Docker
+
+You can always find pre-built images of the backend server at `ghcr.io/grai-io/grai-core/grai-server:latest` and the frontend at `ghcr.io/grai-io/grai-core/grai-frontend:latest`, however, you prefer to build from source you can do so with docker compose.
 
 ```
 git clone https://github.com/grai-io/grai-core
 cd grai-core/grai-server
```

Comment:
Should we reference the example here? we could copy the example to top level and run from there, then have that ignored by source control and lets the user customise as they see fit.
```suggestion
cp examples/deployment/docker-compose/docker-compose.yml .
```